README.md

```diff
@@ -37,7 +37,9 @@ Create a personal [API token](https://circleci.com/docs/2.0/managing-api-tokens/
 ### Contributing
 
 1. Fork it
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+1. Install dev dependencies (`pip install -r requirements-dev.txt`) and `requests` library
```

Comment:
why not add `requests` to dev requirements?

Reply:
So, I didn't initially because it's already in `setup.py`, and I think one is supposed to use `setup.py install` for that based on what I could see?

I've also seen the pattern of generating a `requirements.txt` separately from `requirements-dev.txt` for the non-development deps, but didn't do that either for the reason above (for my projects I've generally used Pipenv, which can be a pain to work with, but does let you define dev and prod dependencies separately, as well as makes caching deps in CI a bit easier).

That said, happy to
 a) Add requests to the `requirements-dev.txt` if you'd like and don't think it's redundant
 b) Set specific version requirements / ranges in the existing dev requirements if you have thoughts / preferences / requirements there.

Reply:
I see no problem with redundancy as long as there is no conflict in `prod` vs `dev` requirements, so having `requests` unpinned in both `setup.py` and dev reqs is fine.

Reply:
sounds good - updated there. removed the `setup.py install` step from the action, since I don't think it's needed anymore.

Successful run in https://github.com/wyardley/pycircleci/actions/runs/1427678004

Reply:
forgot to remove that part `...and 'requests' library.`?

Reply:
good call. updated.